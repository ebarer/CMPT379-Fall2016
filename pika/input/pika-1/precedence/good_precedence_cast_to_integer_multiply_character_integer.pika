exec {

  print [ ^x^ | int ] * 73, _n_.
  print [ ^q^ | int ] * 98, _n_.
  print [ ^n^ | int ] * 66, _n_.
  print [ ^]^ | int ] * 83, _n_.
  print [ ^T^ | int ] * 20, _n_.
  print [ ^@^ | int ] * 39, _n_.
  print [ ^J^ | int ] * 13, _n_.
  print [ ^t^ | int ] * 47, _n_.
  print [ ^!^ | int ] * 72, _n_.
  print [ ^#^ | int ] * 68, _n_.
  print [ ^[^ | int ] * 57, _n_.
  print [ ^u^ | int ] * 34, _n_.

  print 8 * [ ^b^ | int ], _n_.
  print 25 * [ ^x^ | int ], _n_.
  print 91 * [ ^J^ | int ], _n_.
  print 50 * [ ^4^ | int ], _n_.
  print 1 * [ ^\^ | int ], _n_.
  print 71 * [ ^)^ | int ], _n_.
  print 57 * [ ^h^ | int ], _n_.
  print 25 * [ ^|^ | int ], _n_.
  print 68 * [ ^O^ | int ], _n_.
  print 46 * [ ^;^ | int ], _n_.
  print 64 * [ ^k^ | int ], _n_.
  print 21 * [ ^-^ | int ], _n_.

  print [ ^$^ | int ] * [ ^{^ | int ], _n_.
  print [ ^8^ | int ] * [ ^M^ | int ], _n_.
  print [ ^{^ | int ] * [ ^*^ | int ], _n_.
  print [ ^c^ | int ] * [ ^:^ | int ], _n_.
  print [ ^j^ | int ] * [ ^Q^ | int ], _n_.
  print [ ^8^ | int ] * [ ^%^ | int ], _n_.
  print [ ^?^ | int ] * [ ^^^ | int ], _n_.
  print [ ^?^ | int ] * [ ^%^ | int ], _n_.
  print [ ^G^ | int ] * [ ^r^ | int ], _n_.
  print [ ^s^ | int ] * [ ^\^ | int ], _n_.
  print [ ^5^ | int ] * [ ^8^ | int ], _n_.
  print [ ^H^ | int ] * [ ^6^ | int ], _n_.

}

