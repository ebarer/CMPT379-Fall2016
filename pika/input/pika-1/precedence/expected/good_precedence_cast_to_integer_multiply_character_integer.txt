8760
11074
7260
7719
1680
2496
962
5452
2376
2380
5187
3978
784
3000
6734
2600
92
2911
5928
3100
5372
2714
6848
945
4428
4312
5166
5742
8586
2072
5922
2331
8094
10580
2968
3888
